### PR TITLE
Design/nftmarket mini qeury

### PR DESCRIPTION
**Pull request by mkXultra**

### CODE REVIEW

File: x/nftmarket/spec/06_client.md [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+```json
+{
+  "nft":{
+    "id":"a10",
+    "name":"crypotpunk",
+    "description":"crypotpunk is awsome",
+    "symbol":"cryp",
+    "uri":"http...",
+    "uriHash":"xxxxx",
+    "uriHash":"xxxxx",
```
Comment (YasunoriMATSUOKA) [inline x/nftmarket/spec/06_client.md:219]: This is a minor point but there seems to be duplicate.
I guess 1st uriHash means uriHash of class and 2nd uriHash means uriHash of nft, is it right?

**Review by YasunoriMATSUOKA — COMMENTED** [review]
Thanks to this PR I have a better understanding of what data can be used on the front end.
I found only 1 minor point and plz check the comment 🙏

File: x/nftmarket/spec/06_client.md (hunk 1 above)
Comment (mkXultra) [reply x/nftmarket/spec/06_client.md]: I fixed it.

**Comment by mkXultra** [comment]
@YasunoriMATSUOKA 
plz check again.

**Review by YasunoriMATSUOKA — APPROVED** [review]
LGTM!

File: x/nftmarket/spec/06_client.md (hunk 1 above)
Comment (YasunoriMATSUOKA) [reply x/nftmarket/spec/06_client.md]: thank you!

**Pull request merged**